Resources/Private/CustomStylingForCkEditor/src/examplePlugin.js

```diff
@@ -0,0 +1,36 @@
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+import Command from '@ckeditor/ckeditor5-core/src/command';
+
+class ExampleCommand extends Command {
+    execute(callback) {
+        const model = this.editor.model;
+        let ranges = model.document.selection._selection._ranges;
```

Comment:
I am not sure, if you would get the ranges like this @dimaip . I had no chance using getRanges() (always returns empty generator for me). You can directly access to selection and ranges with this . Check `selection.js` in `ckeditor5-engine/model`for further info